src/test/java/seedu/gamebook/logic/commands/EditCommandTest.java

```diff
@@ -99,29 +106,41 @@ public void execute_filteredList_success() {
         assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
     }
 
-    /*
+
     @Test
-    public void execute_duplicateGameEntryUnfilteredList_failure() {
+    public void execute_duplicateGameEntryUnfilteredList_alertsUser() {
         GameEntry firstGameEntry = model.getFilteredGameEntryList().get(INDEX_FIRST_GAMEENTRY.getZeroBased());
         EditGameEntryDescriptor descriptor = new EditGameEntryDescriptorBuilder(firstGameEntry).build();
         EditCommand editCommand = new EditCommand(INDEX_SECOND_GAMEENTRY, descriptor);
 
-        assertCommandFailure(editCommand, model, EditCommand.MESSAGE_DUPLICATE_GAME);
+        try {
+            CommandResult commandResult = editCommand.execute(model);
+            assertEquals(String.format(EditCommand.MESSAGE_EDIT_GAME_SUCCESS, firstGameEntry,
+                    EditCommand.MESSAGE_DUPLICATE_GAME_ENTRY), commandResult.getFeedbackToUser());
+        } catch (CommandException e) {
+            // Should not happen in this case
+        }
     }
-*/
-    /*
+
+
     @Test
-    public void execute_duplicateGameEntryFilteredList_failure() {
+    public void execute_duplicateGameEntryFilteredList_alertsUser() {
         showGameEntryAtIndex(model, INDEX_FIRST_GAMEENTRY);
 
-        // edit game entry in filtered list into a duplicate in gamebook book
+        // edit game entry in filtered list into a duplicate in gamebook
         GameEntry gameEntryInList = model.getGameBook().getGameEntryList().get(INDEX_SECOND_GAMEENTRY.getZeroBased());
         EditCommand editCommand = new EditCommand(INDEX_FIRST_GAMEENTRY,
                 new EditGameEntryDescriptorBuilder(gameEntryInList).build());
 
-        assertCommandFailure(editCommand, model, EditCommand.MESSAGE_DUPLICATE_GAME);
+        try {
+            CommandResult commandResult = editCommand.execute(model);
+            assertEquals(String.format(EditCommand.MESSAGE_EDIT_GAME_SUCCESS, gameEntryInList,
```

Comment:
What's the reason we don't do assertCommandSuccess here?

Reply:
The output message performs a part of this function since we are requiring the output message to be a success message. There should be little need to check whether model updates properly as the other test cases have checked that, and all our manual testing so far verifies that it is alright.